stable/nats/values.yaml

```diff
@@ -218,6 +222,7 @@ cluster:
     ##
     # loadBalancerIP:
 
+
```

Comment:
This line appears

Reply:
fixed in https://github.com/helm/charts/pull/20642/commits/4c1053a9da759afc882d8cc7d709bc47b732171a